src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/config/AwsConfiguration.kt

```diff
@@ -17,19 +19,31 @@ import org.springframework.context.annotation.Primary
 @Configuration
 @ConditionalOnProperty(name = ["offender-events.sqs-provider"], havingValue = "aws")
 class AwsConfiguration(
-  @Value("\${aws.access-key-id}") val accessKeyId: String,
-  @Value("\${aws.secret-access-key}") val secretKey: String,
-  @Value("\${aws.region}") val region: String
+  @Value("\${aws.offender.access-key-id}") val offenderEventsAccessKeyId: String,
```

Comment:
AWS_OFFENDER_EVENTS_ACCESS_KEY is the nname of this variable according to _envs.tpl

Reply:
I'll double check this from the secrets down to the property